package.json

```diff
@@ -6,8 +6,8 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "ncc build -o dist/setup src/setup-java.ts && ncc build -o dist/cleanup src/cleanup-java.ts",
-    "format": "prettier --write **/*.ts",
-    "format-check": "prettier --check **/*.ts",
+    "format": "prettier --write **/*.ts src/distro/*.ts",
```

Comment:
nitpick: I think `src/distro/*.ts` should be covered by ` **/*.ts`?

Reply:
that was what I thought but for some reason prettier wasn't picking it up